document/core/syntax/types.rst

```diff
@@ -54,13 +54,15 @@ Reference Types
 .. math::
    \begin{array}{llll}
    \production{reference type} & \reftype &::=&
-     \FUNCREF ~|~ \EXTERNREF \\
+     \FUNCREF ~|~ \EXTERNREF ~|~ \EXNREF \\
    \end{array}
 
 The type |FUNCREF| denotes the infinite union of all references to :ref:`functions <syntax-func>`, regardless of their :ref:`function types <syntax-functype>`.
 
 The type |EXTERNREF| denotes the infinite union of all references to objects owned by the :ref:`embedder <embedder>` and that can be passed into WebAssembly under this type.
 
+The type |EXNREF| denotes the infinite union of all references to exception :ref:`events <syntax-event>`, regardless of their :ref:`event types <syntax-eventtype>`.
```

Comment:
```suggestion
The type |EXNREF| denotes the a caught exception :ref:`event <syntax-event>`.
```
(there is only one such type, so no union.)
Nit: also, move before externref paragraph.